analyses/filter-mutation-frequencies-tables/filter-mutation-frequencies-tables.Rmd

```diff
@@ -0,0 +1,192 @@
+---
+title: "Filter Mutation Frequencies Tables"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+    toc_depth: 4
+author: Eric Wafula for Pediatric OpenTargets
+date: 01/10/2021
+---
+
+Purpose: Remove Ensembl (ESNG) gene identifier in the mutation frequency tables, including SNV, CNV and fusion that are not in GENCODE v38 and Ensembl package 104.
+
+
+## Usage
+
+To run this from the command line, use:
+```
+Rscript -e "rmarkdown::render('filter-mutation-frequencies-tables.Rmd', clean = TRUE)"
+```
+_This assumes you are in the modules directory of the repository, OpenPedCan-analysis/analyses/filter-mutation-frequency-tables._
+
+## Setup
+
+Load libraries
+```{r}
+# R analysis packages
+suppressPackageStartupMessages(library(data.table))
+suppressPackageStartupMessages(library(tidyverse))
+
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+```
+
+
+Set up directories. 
+```{r}
+# directories for input and output files
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+analyses_dir <- file.path(root_dir, "analyses")
+module_dir <- file.path(analyses_dir, "filter-mutation-frequencies-tables")
+snv_results_dir <- file.path(analyses_dir,  "snv-frequencies/results")
+cnv_results_dir <- file.path(analyses_dir,  "cnv-frequencies/results")
+fusion_results_dir <- file.path(analyses_dir,  "fusion-frequencies/results")
+results_dir <- file.path(module_dir, "results")
+
+# Create results folder if it doesn't exist
+if (!dir.exists(results_dir)) {
+  dir.create(results_dir)
+}
+```
+
+## Get Ensembl gene identifiers
+
+Load Gencode gtf file
+```{r}
+# gtf column names
+col_names <- c("seqname", "source", "feature", "start", "end", "score", "strand", "frame", "attribute")
+
+# read file
+gene_code_annot <- 
```

Comment:
@runjin326, now using `gencode v38` file, `gencode.v38.primary_assembly.annotation.gtf.gz` from the repo data directory. I also updated the gtf loading module from `data.table::fread`  to  `rtracklayer::import`. But `data.table::fread` is faster even though rtracklayer::import` automatically adds column names to the data frame rather than adding them manually as I had done before. Thanks!